test/libsolidity/syntaxTests/bytecodeReferences/library_public_called.sol

```diff
@@ -0,0 +1,10 @@
+library L1 {
+    function foo() internal { new A(); }
+}
+library L2 {
+    function foo() public { L1.foo(); }
+}
+contract A {
+    function f() public { L2.foo(); }
```

Comment:
Why is this not an error?

Reply:
Because `L2.foo()` uses `delegatecall` and thus does not require knowing the bytecode of `L2`.